zenodo/modules/records/ext.py

```diff
@@ -26,14 +26,26 @@
 
 from __future__ import absolute_import, print_function
 
+from werkzeug.utils import cached_property
+import collections
+
 from invenio_indexer.signals import before_record_index
 from invenio_pidrelations.contrib.versioning import versioning_blueprint
+from six import itervalues
+
+from zenodo.modules.records.models import ObjectType
 
 from . import config
 from .custom_metadata import CustomMetadataAPI
 from .indexer import indexer_receiver
 from .utils import serialize_record, transform_record
 from .views import blueprint, record_communities
+from .proxies import current_zenodo_records
+
+
+def current_zenodo_records_ctx():
```

Comment:
Since this is a pretty simple function you can add it as a `lambda: dict(current_zenodo_records=current_zenodo_records)` below